TRANSLATING.md

```diff
@@ -47,6 +47,7 @@ Currently we are aiming to translate the Astro documentation into the following
 - German
 - Japanese
 - Spanish
+- Korean
```

Comment:
Good point! We should also add Polish while we’re here:

```suggestion
- Korean
- Polish
```

Reply:
yes that's also good point! bro! 👍